db/schema.rb

```diff
@@ -11,304 +10,294 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160930190451) do
+ActiveRecord::Schema.define(version: 20161005185412) do
```

Comment:
Be sure to sanitize the schema.rb changes to just make the minimum change, before we merge.